platform/ios/PolyPodApp/Extensions/ColorExtensions.swift

```diff
@@ -27,7 +27,7 @@ extension Color {
         return Double(red * 0.2126 + green * 0.7152 + blue * 0.0722)
     }
     
-    var isLight: Bool { luminance * 255 > 80 }
+    var isLight: Bool { luminance * 255 > 100 }
```

Comment:
??? The first number was kinda arbitrary, but this is, well, still arbitrary. I guess it probably looks good, but to prevent more changes in the future, it would be nice to somehow write the rationale for it somewhere (I'd even admit a comment here)

Reply:
The rationale is just that this value was adjusted based on the UI testing :). But I guess we can employ a better algorithm for determining the luminance.